Common/models/team.model.ts

```diff
@@ -0,0 +1,24 @@
+import {BaseEntity, Column, Entity, Index, PrimaryColumn, PrimaryGeneratedColumn} from "typeorm";
+
+@Entity({ name: "team_model" })
+export default class TeamModel extends BaseEntity {
+
+    @PrimaryGeneratedColumn()
+    teamId: number
+    @Index()
+    @PrimaryColumn("integer")
+    seasonId: number
+    @Column("text")
+    displayName: string
+    @Column("integer")
+    win: number
+    @Column("integer")
+    loss: number
+    @Column("text")
+    abbreviation: string
+    @Column("text")
+    opgg: string
```

Comment:
Just wondering what this is for. We can generate this on the fly.

Reply:
Yea good point, was doing this to 1-1 the sheets